### DISCUSSION TITLE
configuration which will not use bastion/operator servers

### DISCUSSION THREAD

**Discussion by btcdevops-apps** [oracle-terraform-modules/terraform-oci-oke#593] [Q&A]
I want the parameters that I need to disable and enable in order not to use a bastion/operator server where I would have a public Kubernetes API Endpoint.

just setting create_bastion_host and create_operator to false doesn't seem sufficient

**Comment by hyder** [comment]
Apologies for the severely late answer. This is the parameter you need:

`control_plane_type           = "public"`
